Sources/Transaction/TransactionPreparer.swift

```diff
@@ -41,10 +42,13 @@ struct TransactionPreparer {
             Result<Transaction, DefragTransactionPreparationError>
         ) -> Void
     ) {
+        logger.info("")
         guard UInt64.safeCompare(
                 sumOfValues: inputs.map { $0.value },
                 isGreaterThanValue: fee)
         else {
+            logger.debug("failure - sumOfValues inputs: \(redacting: inputs) " +
```

Comment:
We should make this at least `info`, may `warn`?

Reply:
Agreed, I think warning is appropriate, I was gonna make all "failure" related logs warning level, if that sounds good?

Reply:
Sounds good to me.